template/.gitignore.jinja2

```diff
@@ -16,11 +18,13 @@ src/ontology/reports/*
 src/ontology/{{ project.id }}.owl
 src/ontology/{{ project.id }}.obo
 src/ontology/{{ project.id }}.json
+src/ontology/{{ project.id }}.db
 src/ontology/{{ project.id }}-base.*
 src/ontology/{{ project.id }}-basic.*
 src/ontology/{{ project.id }}-full.*
 src/ontology/{{ project.id }}-simple.*
 src/ontology/{{ project.id }}-simple-non-classified.*
+src/ontology/config/context.csv
```

Comment:
I saw your own preferences - I personally like this path here, but tbh I am fine with src/config too - whatever makes more sense. I have used src/ontology/config widely but this will continue to work even if ODK related config appears in `src/config`.

Reply:
> I have used src/ontology/config widely

OK, let’s keep it there then. I’ll fix the `_dynamic_files.jinja2` template accordingly.